src/app/pages/apps/components/app-detail-view/app-details-header/app-details-header.component.scss

```diff
@@ -60,7 +65,6 @@ $app-image-size: 144px;
 
 .app-description {
   flex-basis: 1;
-  max-height: 180px;
```

Comment:
There is a possible case when the app description has long text. It would be nice to trim the description itself when it is long to prevent layout issues.

For example
![image](https://github.com/truenas/webui/assets/351613/c379e146-9738-458c-a43d-938c3620e48e)

Reply:
The ideal pattern IMO would be to have a character limit with a "more" link trigger on the bottom to provide an expanded view.
Perhaps a reusable component or directive for this purpose? I'm sure we could use this type of thing elsewhere in the UI